.github/workflows/verify.yaml

```
@@ -23,13 +23,13 @@ jobs:
 
       - uses: actions/setup-go@cdcb36043654635271a94b9a6d1392de5bb323a7 # v5.0.1
         with:
-          go-version: "1.21"
+          go-version: "1.22"
```

have it use version in go.mod

```suggestion
          go-version-file: go.mod
```